libvirt/tests/src/libvirt_mem.py

```
@@ -116,6 +116,56 @@ def restore_hugepages(page_size=4):
+                    return
+
+                logging.info("No running VMs detected, proceeding with vhost configuration")
+                process.run("modprobe -r vhost_net vhost_scsi vhost_vsock vhost",
+                           shell=True, ignore_status=True)
+                result = process.run("modprobe vhost max_mem_regions=512",
+                                    shell=True, ignore_status=True)
+
+                if result.exit_status == 0:
+                    logging.info("Successfully configured vhost max_mem_regions=512 at runtime")
+                    verify_cmd = "cat /sys/module/vhost/parameters/max_mem_regions"
+                    verify_result = process.run(verify_cmd, shell=True, ignore_status=True)
```

Ruff (0.15.15)

[error] Function call with `shell=True` parameter identified, security issue

(S604)